tutorials/vpn-interop-guides/using-cloud-vpn-with-aws/index.md

```diff
@@ -0,0 +1,591 @@
+
+## IPsec Parameters 
+
+This table covers IPsec parameters to use when configuring VPN gateways and tunnels as described in this document._ _The IPsec connectivity covered in this guide uses the** pre-shared key** generated by AWS for authentication. AWS only supports IKEv1. Detailed GCP-supported IKEv1 ciphers can be found [here.](https://cloud.google.com/vpn/docs/concepts/advanced#supported_ike_ciphers)
+
+<table>
+<thead>
+<tr>
+<th><strong>Parameter</strong></th>
+<th><strong>Value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>IPsec Mode</td>
+<td>ESP+Auth Tunnel mode (Site-to-Site)</td>
+</tr>
+<tr>
+<td>Authentication Protocol</td>
+<td>Pre-shared Key</td>
+</tr>
+<tr>
+<td>Key Exchange</td>
+<td>IKEv1</td>
+</tr>
+<tr>
+<td>Perfect Forward Secrecy (PFS)</td>
+<td>on</td>
+</tr>
+</tbody>
+</table>
+
+# IPsec VPN Using Static Routes
+
+## Reserve an external static IP address for GCP
+
+The AWS VPN configuration requires a remote VPN gateway IP address in advance. Reserve a static external IP address by selecting **External IP addresses** option under the [VPC networks menu](https://console.cloud.google.com/networking/addresses/list) option as shown below.
+
+![image](cloudvpnguide---zv350ow6efi.png)
+
+## Configuration - AWS
+
+For this exercise, create a new VPC network and subnet configuration using the AWS **VPC Wizard** to connect to Google Cloud Platform. The VPC Wizard steps through the creation and configuration of a new VPC network.
+
+### Creating the AWS VPC Network
+
+To get started, log in to the AWS Management Console and select **VPC** from the main services menu.  New AWS accounts all have a default VPC.  
+
+![image](cloudvpnguide---fsaahph6zzd.png)
+
+### 
+
+The first step is to select an IP subnet topology.  There are options for various combinations of private and public IP addressing, with or without VPN connectivity.  Once selected, this configuration cannot be changed.  For this test environment select **Private Subnet Only VPC with Hardware VPN Access**:
+
+![image](cloudvpnguide---0sqb2v4q09i.png)
+
+The next step is to configure the VPC settings:
+
+![image](https://drive.google.com/a/google.com/file/d/1P1yfm98bhnTZUlN0LH7ZF4cBvHIzcEKv/view?usp=drivesdk)
+
+Configure the following required settings:
+
+-  **IP CIDR Block**: this is the CIDR block for the VPC.  It cannot be changed once set.  For this test configuration, enter **10.0.0.0/16.**
+-  **VPC Name**: this is the name of the VPC network.  For this test, enter **GCP-Test**
+-  **Private Subnet**: this is the first subnet allocated from the private IP CIDR block used for AWS services, including EC2.  Enter **10.0.1.0/24 **which is the network on the AWS side that we want to connect to GCP.
+-  **Availability Zone**: this is the AWS Availability Zone into which the VPC will be deployed.  We will leave this set to **no preference.**
```

Comment:
Consider changing the 2nd sentence to "Set this to **no preference**". To match tone of the other steps. Not sure if No Preference should be capitalized.